packages/date-picker/src/vaadin-date-picker-helper.d.ts

```diff
@@ -48,3 +48,11 @@ declare function extractDateParts(date: Date): { day: number; month: number; yea
 declare function dateAfterXMonths(months: number): number;
 
 export { dateAfterXMonths };
+
+/**
+ * Calculate the year of the date based on the provided reference date
+ * Gets a two-digit year and returns a full year.
+ */
+declare function getAdjustedYear(referenceDate: Date, year: number, month: number | null, date: number | null): Date;
```

Comment:
Consider marking `month` and `day` as optional:

```suggestion
declare function getAdjustedYear(referenceDate: Date, year: number, month?: number, day?: number): Date;
```